src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export type Props = {
+  movies?: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
```

Comment:
You could just add a condition for rendering the MoviesList component
```suggestion
export type Props = {
  movies: Movie[];
};

export const MoviesList: React.FC<Props> = ({ movies }) => (
```